web/crux-ui/src/components/registries/edit-registry-card.tsx

```diff
@@ -68,8 +68,10 @@ const EditRegistryCard = (props: EditRegistryCardProps) => {
     onSubmit: async (values, { setSubmitting, setFieldError }) => {
       setSubmitting(true)
 
+      const transformedValues = registrySchema.cast(values) as any
```

Comment:
why we need to cast any?

Reply:
Because `.cast` results in a schema type which is not the same as the model type we need (even though they have the same fields..)

Reply:
made a techdept with low priority
https://dyo.atlassian.net/browse/CRX-392